service/controller/clusterapi/v27/key/cluster.go

```diff
@@ -292,6 +311,38 @@ func baseRoleARN(cluster v1alpha1.Cluster, accountID string, kind string) string
 	return fmt.Sprintf("arn:%s:iam::%s:role/%s-%s-%s", partition, accountID, clusterID, kind, EC2RoleK8s)
 }
 
+func ensureLabel(labels string, key string, value string) string {
+	if key == "" {
+		return labels
+	}
+	if value == "" {
```

Comment:
I think that label with no value is a thing.

Reply:
Don't know. As said, this is just how it works already and I do not want to touch it further now.